src/OSPSuite.SimModelNative/OSPSuite.SimModelNative.vcxproj

```diff
@@ -79,7 +79,7 @@
       <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
-      <DebugInformationFormat>OldStyle</DebugInformationFormat>
+      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
```

Comment:
Figured out that this setting was not required for the Visual Leak Detector. Only "whole program optimization" is relevant. Thus resetting to the previous value.